docs/How-To-Open-a-Homebrew-Pull-Request.md

```diff
@@ -162,3 +186,22 @@ To make changes based on feedback:
 Once all feedback has been addressed and if it's a change we want to include (we include most changes), then we'll add your changes to Homebrew. Note that the PR status may show up as "Closed" instead of "Merged" because of the way we merge contributions. Don't worry: you will still get author credit in the actual merged commit.
 
 Well done, you are now a Homebrew contributor!
+
+## Cleaning up
+
+After your pull request is submitted, you should get yourself back onto `master`, so that `brew update` will pull down new casks properly:
+
+```bash
+# One of:
+cd "$(brew --repository homebrew/core)
+cd "$(brew --repository homebrew/cask)
+# etc
+
+git checkout master
```

Comment:
```suggestion
git -C "$(brew --repository homebrew/core)" checkout master
git -C "$(brew --repository homebrew/cask)" checkout master
```

Reply:
Fixed in 1de9399aa7fb529f1387491ba1359ea71be5f831